extensions/sql-migration/src/dialog/migrationCutover/migrationCutoverDialogModel.ts

```diff
@@ -70,6 +70,18 @@ export class MigrationCutoverDialogModel {
 		return undefined!;
 	}
 
+	public async fetchErrors() {
+		let errors = [];
+		await this.fetchStatus();
+		errors.push(this.migrationOpStatus.error?.message);
```

Comment:
Probably makes more sense to filter out `undefined`s inside this function instead of expecting the caller to remember to do it.

Reply:
This still seems to be pushing possibly-undefined values into the array, I agree with Benjin that it would be good to filter those out before returning.